### ISSUE TITLE
Need to add getAttribute method

### ISSUE THREAD

**Issue by sehgal123** [headspinio/appium-roku-driver#69]
Hi Team, 

I am using appium-roku-driver to automate one of OTT application using Appium with Java. I need to use method getAttribute to get the text, focused etc attributes value for elements on screen. I have added below method in the element.js file and able to use it successfully. 

<img width="566" alt="getAttributeMethod" src="https://github.com/headspinio/appium-roku-driver/assets/50777845/8a98227c-55f9-4506-b080-797b996fa464">

But, while pushing the code on my branch, I am getting below error. Hence, unable to create PR.

remote: Permission to headspinio/appium-roku-driver.git denied to sehgal123.
fatal: unable to access 'https://github.com/headspinio/appium-roku-driver.git/': The requested URL returned error: 403

Please check the issue and provide the access or help to add this method in driver repo.

Thank you!

**Comment by jlipps** [comment]
You don't have access to this repo. The appropriate thing is to fork the repo, push changes to your fork, then propose a PR from the fork.

**Comment by sehgal123** [comment]
Thank you for your response. I have created my fork and create PR - https://github.com/headspinio/appium-roku-driver/pull/70
Please review.

**Comment by sehgal123** [comment]
Hello @jlipps, thank you PR merge. I can see the method in main branch now, but when I reinstalled the roku driver using below command - 
appium driver install --source=npm @headspinio/appium-roku-driver

As newly added method is not there in npm package, with npm cmd installing the driver with previous code. By when, we can get the new release of roku driver.

I also tried installing roku driver with cmd but getting error. 
sudo appium driver install --source=git --package=appium-roku-driver https://github.com/headspinio/appium-roku-driver.git

For now, I have cloned the repo locally and installed via below cmd to test the method.
sudo appium driver install --source=local /Users/Desktop/Roku_Driver/appium-roku-driver
